### DISCUSSION TITLE
Does the compiled wasm can be run out of browsers？

### DISCUSSION THREAD

**Discussion by johnlanni** [i-net-software/JWebAssembly#67] [Q&A]
I saw this description in the readme:
 > The target is to run Java natively in the browser with WebAssembly.

Does this mean that the wasm target can only be run in the browser? I guess the limitation is because there are some browser apis that need to be exported to wasm , is that right?

**Comment by Horcrux7** [comment]
see the [FAQ](https://github.com/i-net-software/JWebAssembly/wiki/FAQ)
